app/javascript/components/payment-error-button.vue

```diff
@@ -2,18 +2,25 @@
   <div class="payment-error-button">
     <modal name="apology-modal">
       <div class="invoice__apology-modal-text">
-        <h1>Error en el pago</h1>
-        Disculpe las molestias :(, lo vamos a revisar para que puedas pagar en el futuro
+        <div v-if="showApologyText">
+          <h1>{{ apologyTextTitle }}</h1>
+            {{ apologyText }}
+            <cartProducts v-if="bla"/>
```

Comment:
Expected indentation of 10 spaces but found 12 spaces                  vue/html-indent